crates/store/re_remote_store_types/proto/rerun/v0/remote_store.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
+
+package rerun.remote_store.v0;
+
+import "rerun/v0/common.proto";
+
+service StorageNode {
+    rpc ListRecordings(ListRecordingsRequest) returns (ListRecordingsResponse) {}
+    rpc Query(QueryRequest) returns (stream QueryResponse) {}
+    rpc GetRecordingMetadata(GetRecordingMetadataRequest) returns (GetRecordingMetadataResponse) {}
+    // TODO (zehiko) - should this be singular recording registration? Currently we can have 1 rrd => many recordings
```

Comment:
Here and everywhere, this should be `TODO(zehiko)`, not `TODO (zehiko)`.
That's important for all of our TODO-based automated tools to work properly.

The fact that this wasn't caught by the CI is a bug -- I'll make a separate PR for that.

Reply:
got it, tnx, fixed.